onnxruntime/core/providers/webnn/builders/impl/conv_op_builder.cc

```diff
@@ -427,6 +427,22 @@ bool ConvOpBuilder::HasSupportedInputsImpl(const Node& node, const WebnnDeviceTy
     return false;
   }
 
+  // WebNN CPU backend (TFLite) only supports default dilations and group.
+  // https://source.chromium.org/chromium/chromium/src/+/main:services/webnn/tflite/graph_builder_tflite.cc;l=1040
+  if (device_type == WebnnDeviceType::CPU && op_type == "ConvTranspose") {
```

Comment:
device_type variable is undefined

Reply:
Arg, I would appreciate less misleading output from the build process.

Reply:
Follow up fix here: https://github.com/microsoft/onnxruntime/pull/21569